ios/README.md

```diff
@@ -1,5 +1,28 @@
-## LibTorch
 
-The PyTorch C++ static library for iOS. 
+## LibTorch for iOS
 
-(Detailed documentation  will be added soon)
\ No newline at end of file
+### Cocoapods Developers
+
+PyTorch is now available via Cocoapods, to integrate it to your project, simply add the following line to your `Podfile` and run `pod install`
+
+```ruby
+pod 'LibTorch'
+```
+
+### Import the library
+
+For Objective-C developers, simply just import the umbrella header
+
+```
+#import <LibTorch/LibTorch.h>
+```
+
+For Swift developers, you need to create an Objective-C class as a bridge to call the C++ APIs. We highly recommend you to follow the [Image Classification](https://github.com/pytorch/examples) demo where you can find out how C++, Objective-C and Swift work together. 
```

Comment:
Can you update this to the URL where your iOS demo will go?

Reply:
The demo app hasn't been landed, I'll update the link later.